Voice of Time/Transfer/CSocketHold.cs

```diff
@@ -182,17 +182,32 @@ private async Task QueueHandler()
                     if (isCancelled) return;
                     var nextQueueItem = Queue.Dequeue();
 
-                    var messageBytes = Encoding.UTF8.GetBytes(nextQueueItem.Message + Constants.EOM);
+                    var messageBytes = Encoding.UTF8.GetBytes(Constants.SOM + nextQueueItem.Message + Constants.EOM);
                     var code = await Client.SendAsync(messageBytes, SocketFlags.None);
 
                     bool messageComplete = false;
                     string IncomingMessage = "";
-                    while (!messageComplete)
+
+
+                    //-----------------------------------------------------------------------------
+
+                    //string IncomingMessage = ""; // <- + Erster Teil der Respnse
+
+                    var buffer = new byte[Constants.BUFFER_SIZE_BYTE];
+                    var received = await Client.ReceiveAsync(buffer, SocketFlags.None);
+                    var response = Encoding.UTF8.GetString(buffer, 0, received);
+
+                    var indexOfSOM = response.IndexOf(Constants.SOM);
+
+
+                    if (indexOfSOM > -1)
                     {
-                        var buffer = new byte[Constants.BUFFER_SIZE_BYTE];
-                        var received = await Client.ReceiveAsync(buffer, SocketFlags.None);
-                        var response = Encoding.UTF8.GetString(buffer, 0, received);
+                        response = response.Remove(indexOfSOM);
+                    }//(Wenn SOM nicht vorhanden: Kommunikation abbrechen)
+
 
+                    while (!messageComplete)
```

Comment:
Was ist die Funktion dieser schleife? Entweder endet diese in einem Enlossloop oder wird nur einmal durchlaufen, abhängig ob ein EOM token sich in der Nachricht befindet